### ISSUE TITLE
"other than" meaning 'apart from'

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#431]
https://universal.grew.fr/?custom=650f8f5e94c4a

**Comment by nschneid** [comment]
1. I have other ideas than this one
2. I have ideas other than this one
3. Other than this one, I have more ideas

Arguably, "other" is an ADJ and functions as `amod` in (1) and (2). But it's not really attributive in (3)—it functions more like `case` (a deadjectival connective).

(The part after "than" may also be clausal: "I have other ideas than repaving the driveway", etc.)

Related: "apart from", "except for"—but "apart" and "except" aren't ADJs

Better analogies: "separate/distinct from"? Or "more than": "More than for the money, he did it for the prestige."

**Comment by amir-zeldes** [comment]
I agree it's just a potentially extraposed ADJ/amod in 1.-2. The preposed version is adverbial as a whole, and ON mixes JJ and RB here. I would be open to having other/RB in 3., but I think it's good to keep a consistent structure for its dependents, so I would say it's still the head (not case) in 3., and takes an obl/advcl child with "than" as case/mark.

**Comment by nschneid** [comment]
I dunno, I think it really behaves like `case` in (3) at least. Its function is to mark a sort of concessive or setting-aside adjunct whose main content is "this one". It lacks the 'additional' meaning present in (1). (Note that for (3) to be a paraphrase I had to add "more". "Other than this one, I have ideas" would not quite work.) In terms of how it marks a nominal, can't think of a salient difference from "except for" (it's just that "except" can't be an adjective). "Other" cannot appear on its own, as a typical `advmod` in a clause might. The combination "other" + "than" is entrenched and thus arguably `fixed` along the lines of "prior to". (We never resolved whether "except for" would be better as `fixed`: see UniversalDependencies/docs#795.)

If you don't like `fixed`, another option is to allow ADJs as `case` like we do for VERBs, and have it be sister to "than".

I hear you that there's a cline of grammaticalization here (rather than a sharp reanalysis) and thus it would be nice to keep the structure similar, but I think the content-heads policy prevents us from doing that as far as items grammaticalizing as `case`/`mark` are concerned. (By contrast, in PTB or CGEL, prepositions would be heads as well so the structure wouldn't have to change.) IOW, the content-heads policy prevents continuity of structure as items progress from content to functional.

**Comment by amir-zeldes** [comment]
> the content-heads policy prevents us from doing that 

I think that presupposes that "other" is not a content head :) I understand what you are saying about grammaticalization messing with consistency, but I don't think it's necessary in this case. "Other than this one" really does mean that there are other ideas (it is pretty literally connected to the meaning of 'other'), and it has the regular argstr of other + than. If you treat it as functional, would you also treat "similarly to" as case? I think it's pretty similar, and we could decide that it's a complex case/mark, but I don't see what we would gain by doing that. It's meaning, like "other", is pretty close to a preposition, since they are both relational adjectives.

**Comment by nschneid** [comment]
"Similarly" can occur on its own as an adjunct in a clause.

"Other (than)" can't, and also can't be premodified when it's marking an adjunct:
- That is a whole other problem than what I was talking about.
- *That is a problem whole other than what I was talking about.
- *Whole other than what I was talking about, that is a problem.

I do think the meaning is a bit different (albeit related). The adjunct use doesn't have to occur with an explicit reference to something of which there are multiple:
- Other than the problem we had last week, the project has been running smoothly.
   * Like _aside from_/_apart from_/_except for_, this doesn't imply there are multiple problems. Whereas "Apart from the **other** problem we had last week, the project has been running smoothly" DOES imply multiple problems.

**Comment by amir-zeldes** [comment]
> "Other (than)" can't

I think that's just because "other" doesn't take "ly" morphologically. It's corresponding adverbial form is "otherwise", which we also treat as `advmod` and not `mark`:

  * Similarly to mice, bats have fur
  * Similar to mice, bats have fur
  * Other than bats, there are no flying mammals
  * Otherwise, there are no flying mammals

So the form "similar" corresponds to "other" in not being used alone in this position (?? similar, bats have fur). The 'multiple reference' thing I think just has to do with the meaning of 'other' compared to 'similar', and I don't think that should change the morphosyntactic classification - they are both relational, and both express a relatum argument.

**Comment by nschneid** [comment]
How would you parse

- Similar to your distaste for coffee, I can't stand cilantro.

**Comment by nschneid** [comment]
Also, "prior to" and "such as" are in the fixed list. Is "other than" different in some crucial way?

**Comment by amir-zeldes** [comment]
> Similar to your distaste for coffee, I can't stand cilantro.

I think you'd need to decide whether you consider it a reduced clause (advcl) or just a short-form adverb and treat it like "similarly", but either way I wouldn't want it to be a preposition. It is pretty transparently the usual meaning and argstr of "similar".

> "prior to" and "such as" are in the fixed list

If I had made that list, prior to would not be in it - it's just an adverb with an argument. "such as" is quite far from it's original meaning and distro (people don't really use "as" by itself for comparison of NP), so I would support that being `fixed`. More generally though, I think we should keep `fixed` as small as possible, it's already overly lax for my taste.

**Comment by nschneid** [comment]
I agree that "similar to" is a weaker case for `fixed` (for one I think it can be modified: "Very similar to your distaste for coffee,..."), but I'm puzzled by how it seems to behave like "similarly" here.

There's also things like "Reminiscent of" and "Contrary to". Such expressions are sometimes "detached" from a particular predicand and act more like discourse connectives.

I thought "Mindful/conscious/cognizant of" might have similar properties, acting as a speaker-oriented adverbial, but glancing at COCA, it seems these are usually followed by a predicand: "Mindful of the league's difficulties, **Toliver** has a singular mission..."

Regarding the fixed list in general: its composition seems to be a bit ad hoc, I think we really need some better principles for what should be in vs. out. Maybe "prior to" should be removed, but I would be more comfortable if there were a clear test. This seems like something the UniDive MWE folks should be able to help us with.

**Comment by amir-zeldes** [comment]
Yeah, I don't disagree with any of that, those are all valid points. 

On a practical level it seems to me that whatever we do, there will be items on the fence, so my preference has always been to follow the maxim "pick one bad annotation scheme and stick to it". What I really don't like are frequent changes, and given that the fixed list has been around for UD English for a long while, I'm happy to live with its quirks just like I live with PTB quirks, but with the confidence that it's stable and I have a good idea of what's in there - then I can compare versions through time, compare different datasets "not built here" etc. For English upos, for example, I have very little confidence I can compare any two datasets out there in detail, but I'm more confident about the fixed list because it's been so stable across corpus/parser iterations.

**Comment by nschneid** [comment]
There was an older issue on this: #275

**Comment by nschneid** [comment]
It occurred to me to check what PTB does. They treat "other" as an adjective heading an ADVP:

<img width="593" alt="image" src="https://github.com/UniversalDependencies/UD_English-EWT/assets/985263/65cd4d4d-ab17-411a-9feb-a8dd753793c1">

It doesn't feel like a separate clause to me (unlike resultative secondary predicates, for example).

It turns out that an ADJ is allowed to attach as `advmod`, and this happens in the data, though a lot of these look like tag or tree errors: [EWT](https://universal.grew.fr/?custom=65242e3ddc9be), [GUM](https://universal.grew.fr/?custom=65242eb4c15c4)

So what about

- Other than bats, there are no flying mammals: advmod(are, Other/ADJ), obl(Other, bats)?

A few of the EWT examples have "other" as ADV/RB, but it's not consistent.
